changelog.d/20240711_190719_yadudoc1729_mpi_engine_2.rst

```diff
@@ -0,0 +1,37 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Adding `GlobusMPIEngine` with better support for MPI applications.
+  `GlobusMPIEngine` uses Parsl's `MPIExecutor` under the covers to
+  dynamically partition a single batch job to schedule MPI tasks.
+
+  Here's an example endpoint configuration that uses `GlobusMPIEngine`
```

Comment:
We may want to add this to our example endpoint configuration docs.

Reply:
Agreed, I've added the MPI config for expanse to the endpoint configuration examples page. For the rest of the machines it'll need work get tested configs into the docs. The plan was to add a more complete docs pages along with `MPIFunctions` since we need that piece to properly make use of `GlobusMPIEngine`.